README.md

```diff
@@ -58,55 +58,143 @@ documentation](https://www.packer.io/docs).
    of the VM to run automated tests.
 
 
-# The last step
+## The last part first (overview step 3)
 
-Assuming all you need to do is tweak the package list, add or adjust a makefile
-target, start here.  Before you go deleting all the seemingly "unnecessary" and
-"extra" packages, please remember these VM images are shared by automation
-in multiple repositories :D
+a.k.a. ***Cache Images***
 
-1. VM configuration starts with one of the `cache_images/*_setup.sh` scripts.
+These are the VM Images actually used by other repositories for automated
+testing.  So, assuming you just need to update packages or tweak the list,
+start here.  Though be aware, this repository does not thrroughly test images
+in their native-habitat, that's your secondary responsibility (see below).
```

Comment:
I can't figure out what this means ("in their native habitat"), could you rephrase? And can you provide an explicit link such as "see FOO below"?

Reply:
Meaning, it doesn't run podman tests on them.

Reply:
in fact, right now it doesn't run *any* tests :confused:

Reply:
Have you double-checked your YAML? The `<<:` thing inside `matrix` in `base_images` and `cache_images` does not seem to do what you think it does. (Discovered because I got way too lost reviewing this, and had to convert the cirrus.yml to actual-yaml to use my graph tool, which then drew nonsensical results, which turned out to be due to the `<<` things)